resources/design-document.md

```diff
@@ -148,31 +148,33 @@ String medInfo;
 
 
 ## 6.6 _Get Notification Endpoint_
-* Accepts `GET` requests to `/notifications/:customerId`
-* Scans medication table based on customerId and time, returns all Notifications for that customerId at that time window of 15 minutes before and 15 minutes after that time.
+* Accepts `GET` requests to `/notifications/:time` 
+* Query notification gsi if the parameter passed in is time, returns all Notifications for that customerId at that time window of 15 minutes before and 15 minutes after that time. This is used to populate the banner only, not the health chart. 
     * If there are no notifications on the table, return an empty Set.
 
 ![Sequence Diagram Get Notifications.png](images%2FdesignImages%2FSequence%20Diagram%20Get%20Notifications.png)
 
-
 ## 6.7 _Add Notification Endpoint_
 
-* Accepts `POST` requests to `/notifications/:customerId`
-* Accepts a customer ID and a time to be added.
+* Accepts `POST` requests to `/notifications/:medName/time`
+* Accepts a customer ID, medName and a time to be added.
```

Comment:
The comment above about POST URIs (and body) applies here, too. (Details needed to create the new notification should be passed in the body of the request, instead of the path.)

Reply:
Oh ok gotcha. So the path would just be /notifications/:medName and then in the body would be the time?